README.md

```diff
@@ -175,8 +178,56 @@ ___
 
  ## SNP Data Processing
 
-Currently under construction
+lusSTR is able to process SNPs derived from the ForenSeq Signature Prep assay and the ForenSeq Kintelligence assay. SNPs from the ForenSeq Signature Prep assay could be analyzed using either the Verogen UAS or STRait Razor. SNPs from the ForenSeq Kintelligence assay must first be analyzed using the UAS.
+
+___
+### Creating the STR config file
```

Comment:
Copy/paste error: should be "Creating the SNP config file"

Reply:
lol good catch